meta-balena-common/recipes-extended/procps/procps_%.bbappend

```diff
@@ -0,0 +1,24 @@
+
+ALTERNATIVE_${PN} = ""
+
+python procps_binpackages () {
+    def pkg_hook(f, pkg, file_regex, output_pattern, modulename):
+        pn = d.getVar('PN')
+        # d.appendVar('RRECOMMENDS_%s' % pn, ' %s' % pkg)
```

Comment:
This needs to be uncommented. RRECOMMEND packages (if any) should also be split.

Reply:
I had some fundamental mistakes at this point, where I couldn't RRECOMMEND packages because each package had a dependency on the base package. So including one would include them all.

I since fixed it up to behave more like util-linux where the base PN is actually an empty meta-package and each sub package only depends on `procps-libprocps`.